File: Multiplesof3and5/M35.java
/*
Project Euler # 1

Brute Force
*/

public class M35 {
    public static void main(String[] args) {
        int sum = 0;

        for ( int i=0; i<1000; i++ )
            if ( i % 3 == 0 || i % 5 == 0 )
                sum += i;

        System.out.println(sum);
    }
}

File: Multiplesof3and5/M35.js
/*
Project Euler # 1

Brute Force
*/

var lim = 1000;
var sum = 0;

for (var i = 0; i < lim; i++) {
    if ( i % 3 === 0 || i % 5 === 0) {
        sum += i;
    }
}

console.log(sum);
